src/main/java/dev/j3fftw/litexpansion/LiteXpansion.java

```diff
@@ -36,6 +37,16 @@ public void onEnable() {
         if (!new File(getDataFolder(), "config.yml").exists())
             saveDefaultConfig();
 
+        if (!getConfig().contains("sprinkler.success-chance")) {
+            getConfig().set("sprinkler.success-chance", 0.5);
+            saveConfig();
+        }
+
+        if (!getConfig().contains("sprinkler.particles")) {
+            getConfig().set("sprinkler.particles", true);
+            saveConfig();
+        }
+
```

Comment:
like cookie mentioned make this a item config

Reply:
What does item config mean?

Reply:
items.yml  like where you can change item settings